Resources/Prototypes/Loadouts/Jobs/Security/uncategorized.yml

```diff
@@ -926,6 +944,25 @@
   items:
     - WeaponRevolverArgentiNonLethalSecurity
 
+- type: loadout
+  id: LoadoutSecurityEnergyRevolver
+  category: JobsSecurityAUncategorized
+  cost: 10
+  canBeHeirloom: true
+  guideEntry: SecurityWeapons
+  requirements:
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSecurityWeapons
+    - !type:CharacterDepartmentTimeRequirement
+      department: Security
+      min: 180000 # 50 hours (veterancy bonus)
+    - !type:CharacterDepartmentRequirement
+      departments:
+        - Security
+    - !type:CharacterAgeRequirement
+      min: 21
+  items:
+    - WeaponEnergyRevolver
```

Comment:
For consistency's sake, add a `WeaponEnergyRevolverSecurity` weapon that inherits from `WeaponEnergyRevolver` that has `The serial number on the handguard marks this gun as belonging to an NT Security Officer.` appended to its description, a suffix to distinguish it from the regular Energy Revolver in the Entity Spawn Panel, and a link to the guide on security weapons, and use that on the loadout entry instead.

Example:
https://github.com/Simple-Station/Einstein-Engines/blob/09b5c3ed9da1f6b51a866ec1e116aa12ddceecab/Resources/Prototypes/Entities/Objects/Weapons/Guns/Pistols/pistols.yml#L125-L134